pom.xml

```diff
@@ -164,7 +164,7 @@
         <phoenix.version>4.7.0.2.5.0.0-1245</phoenix.version>
         <redis.lettuce.version>3.4.2.Final</redis.lettuce.version>
         <scala.version>2.10.5</scala.version>
-        <schema-registry.version>0.5.0</schema-registry.version>
+        <schema-registry.version>0.5.1.3.1.1.0-6</schema-registry.version>
```

Comment:
This will be changed to 0.5.1 once it is released(plan to do that by this weekend viz 12th Feb)